incubator/couchdb/templates/statefulset.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: {{ template "couchdb.fullname" . }}
   labels:
     app: {{ template "couchdb.name" . }}
-    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    chart: {{ .Chart.Name }}
```

Comment:
Don't change labels on statefulset. There's already a correct `spec.selector`.

Reply:
Done, kept the version label here